client/src/RestaurantList/RestaurantList.jsx

```diff
@@ -0,0 +1,35 @@
+// @flow
+import React, { Component } from 'react';
+import { RestaurantCard } from './RestaurantCard';
+import type { Restaurant } from '../flowTypes';
+
+type State = Array<Restaurant>
+
+class RestaurantList extends Component<{}, State> {
```

Comment:
IMO change the component to get restaurants as props

Reply:
Will be handled when I'll integrate state management :)